lib/api/CInferenceModelDefinition.cc

```diff
@@ -102,6 +113,10 @@ auto toJson(std::size_t value, CSerializableToJson::TRapidJsonWriter&) {
     return toJson(value);
 }
 
+auto toJson(const unsigned int& value) {
```

Comment:
If this is for handling `uint32_t` then be aware this might be fragile for portability in the future.  On Windows `uint32_t` might be `unsigned long`, as on 64 bit Windows `long` is still 32 bits like `int`.

Reply:
Indeed, I don't need it after refactoring.